python-lib/dku_google/clusters.py

```diff
@@ -304,9 +299,6 @@ def build(self):
                 # assume it's an existing range name (shared VPC case)
                 ip_allocation_policy["clusterSecondaryRangeName"] = cluster_pod_ip_range
             create_cluster_request_body["cluster"]["ipAllocationPolicy"] = ip_allocation_policy
-
-        if self.legacy_auth:
-            create_cluster_request_body["cluster"]["legacyAbac"] = {"enabled":True}
             
         need_issue_certificate = False
```

Comment:
Maybe that part can be removed too (L303-L318) as we won't be using the certificates anymore?